README.md

```diff
@@ -39,6 +39,22 @@ assert o.unwrap_or_else(str.upper) == 'yay'
 assert n.unwrap_or_else(lambda: 'default') == 'default'
 ```
 
+There are some methods that support conversion from a `Maybe` to a `Result` type
+in the [result library](https://github.com/rustedpy/result/).  If you wish to
+leverage these methods, you must add `result` to your dependency list:
```

Comment:
Given the number of changes that have been done in the CI config, project config (setup.cfg, makefile), etc, I'm wondering if it's time to merge the two repos and libraries into a single one. It will make it easier to keep the two repo's setup, config and structure in sync. Just thinking out loud, this PR can continue as it is, it's something I can think about more afterwards.

Reply:
I was wondering the same thing, but I didn't want to ask because I know I saw another issue elsewhere where you had discussed it with somebody else. I certainly would support such a decision. Would you create a new repo since the name `result` is perhaps too narrow to sensibly include `maybe` as well? If so, any idea on what you might name the new repo? Perhaps just `rustedpy`?

Reply:
Not sure, name is probably the hardest part.
That or single repo, but publish to separate pypi packages.